apollo-router/src/axum_http_server_factory.rs

```diff
@@ -99,12 +99,55 @@ pub(crate) struct ListenersAndRouters {
 pub(crate) fn make_axum_router<RF>(
     service_factory: RF,
     configuration: &Configuration,
-    endpoints: MultiMap<ListenAddr, Endpoint>,
+    mut endpoints: MultiMap<ListenAddr, Endpoint>,
 ) -> Result<ListenersAndRouters, ApolloRouterError>
 where
     RF: SupergraphServiceFactory,
 {
-    let mut main_endpoint = main_endpoint(service_factory, configuration)?;
+    if !sandbox_on_main_endpoint(configuration) {
+        endpoints.insert(
+            configuration.sandbox.listen.clone(),
+            Endpoint::new(
+                configuration.sandbox.path.clone(),
+                service_fn(|_req: transport::Request| async move {
+                    Ok::<_, BoxError>(
+                        http::Response::builder()
+                            .header("Content-Type", "text/html")
```

Comment:
Could you use https://docs.rs/http/latest/http/header/constant.CONTENT_TYPE.html ?

Reply:
Sure!

Reply:
fixed in [72a7b81](https://github.com/apollographql/router/pull/1718/commits/72a7b81693a3e9c829155b96ff41e4d93f0d9ed3)